Firebase/Auth/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v4.5.1
+- Adds client-side validation to prevent setting handleCodeInApp to false when performing
```

Comment:
nit: code segments should be backtick-escaped. 

```
`handleCodeInApp`
```

Reply:
Done, thanks.